### PR TITLE
Added Short.cm support

### PR DESCRIPTION
**Pull request by Kamleeonau**

### CODE REVIEW

File: pyshorteners/shorteners/shortcm.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        Returns:
+            A string containing the expanded URL
+
+        Raises:
+            ExpandingErrorException: If the API Returns an error as response
+        """
+        expand_url = f"{self.api_url}expand"
+
+        # split domain and path
+        url_parser = re.compile("(https?://)([^/]*)/([^/?]*)")
```
Comment (ellisonleao) [inline pyshorteners/shorteners/shortcm.py]: how about using [urllib.parse](https://docs.python.org/3/library/urllib.parse.html) here?

Comment (Kamleeonau) [reply pyshorteners/shorteners/shortcm.py]: I've pushed a new commit with unit testing and changed out the regex matching for urllib.parse as suggested (thanks for that btw)

Comment (ellisonleao) [reply pyshorteners/shorteners/shortcm.py]: i cant see that change.. can you check if it was actually pushed please?

Comment (ellisonleao) [reply pyshorteners/shorteners/shortcm.py]: ok now its there

**Pull request merged**